content/05.main-text.md

```diff
@@ -159,6 +165,18 @@ while doing so in an unsupervised manner that captures the semantic richness of
 
 ## Results {.page_break_before}
 
+### Hetio Website
+
+We registered the domain `https://het.io` and built a website hosted at that address to serve as a unified, publicly-palatable home for Hetio.
+
+Primarily, the website is where the connectivity search webapp can be used, as well as several other interactive apps.
+It also includes high level information on hetnets and Hetionet, citation and contact details, links to supporting studies and software, downloads and exploration of Hetionet data, and various Hetio-related media.
+
+![
+**Homepage of the Hetio website.**
+Provides a succinct overview of what Hetionet consists of and what its purpose is.
+](https://github.com/greenelab/connectivity-search-manuscript/raw/b290b4ad435553c7126867e0720f3112b4692809/content/media/website/website-homepage.png){#fig:website ) and other miscellaneous logos and iconography across the websiteidth="100%" .white}
```

Comment:
Looks like syntax is weird at `{#fig:website ) and other miscellaneous logos and iconography across the websiteidth="100%" .white}`

For the figure URL, you can use this one for now that uses the commit hash from this PR:

```
https://github.com/greenelab/connectivity-search-manuscript/raw/d5766924e8c774accdc143bea352e49610ee0673/content/media/website/website-homepage.png
```